src/librawspeed/metadata/ColorFilterArray.h

```diff
@@ -27,16 +27,15 @@
 namespace RawSpeed {
 
 enum CFAColor {
-  CFA_COLOR_MIN = 0,
+  // see also DngDecoder
   CFA_RED = 0,
   CFA_GREEN = 1,
   CFA_BLUE = 2,
-  CFA_GREEN2 = 3,
+  CFA_INVALID = 3, // keep the unit tests happy
```

Comment:
Well, I think the enum should completely go away but your turn-around-test requires to have something mapping to 3. I chose "invalid" just to prevent someone from later thinking "ah GREEN2 sounds nice, I take that" and then reintroducing the name in the code again.

The proper fix would be to change the interface and separate the two purposes you mentioned. But you expressed your explicit worries about changing anything in that regard, so I just fixed #34 here.